pytripgui/plan_executor/patient_model.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import pytrip as pt
+from pytrip.tripexecuter import Plan
+from pytrip.tripexecuter import Field
+
+logger = logging.getLogger(__name__)
+
+
+class PatientModel:
+    def __init__(self):
+        self.name = "Patient"
+        self.ctx = None
+        self.vdx = None
+
+        self.plans = []
+        self.simulation_results = []
+
+    def open_ctx(self, path):
+        ctx = pt.CtxCube()
+        ctx.read(path)
+        self.ctx = ctx
+        self.name = ctx.basename
+
+    def open_vdx(self, path):
+        vdx = pt.VdxCube(self.ctx)
+        vdx.read(path)
+        self.vdx = vdx
+        if self.name != vdx.basename:
+            logger.error("CTX | VDX patient name not match")
```

Comment:
This should possibly be a window error message.

Reply:
In the future, I'd like to push everything from this folder (plan_executer) to pytrip. Now I develop it in pytripgui, but as you can see all files in this folder don't import GUI stuff. So instead of making window here can I throw exception for example?

Reply:
I agree with @ljelen that on this level a well designed exception is better solution, but provided that in some other place of the application (where?) it is catched and displayed as a "windows error message".